pkg/seccomp/default_linux.go

```diff
@@ -44,8 +44,54 @@ func arches() []Architecture {
 // DefaultProfile defines the allowlist for the default seccomp profile.
 func DefaultProfile() *Seccomp {
 	einval := uint(unix.EINVAL)
+	enosys := uint(unix.ENOSYS)
+	eperm := uint(unix.EPERM)
 
 	syscalls := []*Syscall{
+		{
+			Names: []string{
+				"bdflush",
+				"clone3",
```

Comment:
I think this has to be treated the same way as `clone`. Ideally, we would allow this system call because it is required for supporting certain forms of security hardening. But we definitely will need `ENOSYS` for `clone3`.